src/gui/transferlistwidget.cpp

```diff
@@ -574,6 +574,18 @@ void TransferListWidget::copySelectedIDs() const
     qApp->clipboard()->setText(torrentIDs.join(u'\n'));
 }
 
+void TransferListWidget::copySelectedComments() const
+{
+    QStringList torrentComments;
+    for (const BitTorrent::Torrent *torrent : asConst(getSelectedTorrents()))
+    {
+        if (!torrent->comment().isEmpty())
+            torrentComments << torrent->comment();
+    }
+
+    qApp->clipboard()->setText(torrentComments.join(u"\n---------\n"_s));
```

Comment:
One comment can span multiple lines and even contain empty lines. Is something like this acceptable or even desirable to differentiate comments from each other?

Reply:
Lets use it until someone has a better idea.